src/LibCURL.jl

```diff
@@ -21,27 +14,63 @@ function __init__()
     global cacert = MozillaCACerts_jll.cacert
 end
 
-include("lC_exports_h.jl")
-include("lC_common_h.jl")
-include("lC_curl_h.jl")
+@static if Sys.islinux() && Sys.ARCH === :aarch64 && !occursin("musl", Base.BUILD_TRIPLET)
+    include("wrappers/aarch64-linux-gnu.jl")
```

Comment:
These should probably live inside of `deps/`

Reply:
I changed the folder to `lib` like what's being done in https://github.com/maleadt/LLVM.jl.